src/components/searchBar/index.tsx

```diff
@@ -1,23 +1,96 @@
 import React, { useCallback, useState } from "react";
 
+interface IngredientProps {
```

Comment:
We can simplify the interface name here
```suggestion
interface Ingredient {
```

Reply:
I like the idea to have this standard to create the interfaces, is that ok?

Reply:
We can proceed with this naming convention but consider the following:

1. this codebase might grow a lot in the future. This means that adhering to a certain naming convention can greatly improve readability in the long run (your suggestion is great with regards to making this consistent)
2. react components utilise props (widely accepted naming convention) which leads me to believe that this `interface` is describing component props and not an ingredient entity. What is the planned naming convention for component props?
3. ideally, we wish to name stuff in such a way that when a new engineer joins the team, he can figure out stuff with minimal cognitive effort (point above has a probable assumption someone might have when reading the code the first time)
4. when talking about the ingredient entity, naming the interface `Ingredient` doesn't seem misleading in any way (at least to me) and is succinct (again this is my personal preference). If we wish to be verbose, we can also go with `IngredientEntity` and then we can differentiate an ingredient entity type from a ingredient component props type without the need to look around the code much.

Reply:
Although I have this personal preference, I was taking a look on the official Typescript interface page and all the examples I saw that were just using the component name, so I suppose that I should follow this pattern ( I haven't seen any other naming convention). If all that you said in mind I prefer to use only Ingredient rather than IngredientEntity, it makes more sense for me.

Reply:
Sure, let's rename it then.